TiKV config: update warnings about `enable-ttl` (#8856)
pingcap/docs-cn: tikv-configuration-file.md

### `enable-ttl`

> **警告：**
>

> > - 你**只能**在部署新的 TiKV 集群时将 `enable-ttl` 的值设置为 `true` 或 `false`，不可以在已经存在的 TiKV 集群中修改该配置项的值。由于该配置项为 `true` 的和 `false` 的 TiKV 集群所存储的数据的格式互不相同，如果你在已经存在的 TiKV 集群中修改该配置项的值，会造成不同格式的数据存储在同一个集群，导致重启对应的 TiKV 集群时 TiKV 报 "can't enable ttl on a non-ttl instance" 错误。

Reviewer: ```suggestion
> - 你**只能**在部署新的 TiKV 集群时将 `enable-ttl` 的值设置为 `true` 或 `false`，**不能**在已有的 TiKV 集群中修改该配置项的值。由于该配置项为 `true` 和 `false` 的 TiKV 集群所存储的数据格式不相同，如果你在已有的 TiKV 集群中修改该配置项的值，会造成不同格式的数据存储在同一个集群，导致重启对应的 TiKV 集群时 TiKV 报 "can't enable ttl on a non-ttl instance" 错误。
```

> > - 你**只能**在 TiKV 集群中使用 `enable-ttl`，不可以在有 TiDB 节点的集群中使用该配置项（即在此类集群中把 `enable-ttl` 设置为 `true`），否则会导致数据损坏、TiDB 集群升级失败等严重后果。

Reviewer: ```suggestion
> - 你**只能**在 TiKV 集群中使用 `enable-ttl`，**不能**在有 TiDB 节点的集群中使用该配置项（即在此类集群中把 `enable-ttl` 设置为 `true`），否则会导致数据损坏、TiDB 集群升级失败等严重后果。
```

### `enable-ttl`

> **警告：**
>

> > - TTL 暂时只适用于 RawKV 接口。由于所涉及底层数据格式不同，你只能在新建集群时设置该功能。

Reviewer: 不太明白“所涉及底层数据格式不同”是什么意思。不同的两个主体分别是什么呢？

Reviewer 2: @TomShawn 这个配置是用来设置数据可以在 xx 时间后被清理。所以开启与否，对于目前 tikv 中存储的数据格式是不一样的。

> - **禁止**在已有集群上修改该项配置，否则会导致启动报错。

> > - **禁止**在有 TiDB 节点的集群中使用该参数，否则会导致数据损坏、TiDB 集群升级失败等严重后果。

Reviewer: 无 TiDB 节点的集群是什么集群呢？TiKV 集群吗？

Reviewer 2: @TomShawn 是的，没有 tidb 节点的集群。

> **警告：**
>
> - TTL 暂时只适用于 RawKV 接口。由于所涉及底层数据格式不同，你只能在新建集群时设置该功能。
> - **禁止**在已有集群上修改该项配置，否则会导致启动报错。

> > - **禁止**在有 TiDB 节点的集群中使用该参数，否则会导致数据写坏、TiDB 集群升级失败等严重后果。

Author: ```suggestion
> - **禁止**在有 TiDB 节点的集群中使用该参数，否则会导致数据损坏、TiDB 集群升级失败等严重后果。
```
